index.js

```diff
@@ -497,18 +498,24 @@ module.exports = function (log, indexesPath) {
     const indexNeedsUpdate = !coreIndexNames.includes(op.data.indexName)
 
     function save(count, offset) {
-      if (updatedSeqIndex) saveCoreIndex('seq', indexes['seq'], count)
+      const coreWrites = []
```

Comment:
How about multicb instead? In this file we already use multicb (and that's because it was faster than Promise.all) and there are so far zero promises in this file.

Reply:
*I can't promise* (pun intended)

Reply:
Good call :)